share/doc/homebrew/Maintainers-Avoiding-Burnout.md

```diff
@@ -0,0 +1,29 @@
+# Maintainers: Avoiding Burnout
+**This guide is for maintainers.** These special people have **write
+access** to Homebrew’s repository and help merge the contributions of
+others. You may find what is written here interesting, but it’s
+definitely not for everyone.
+
+# 1. Use Homebrew
+
+Maintainers of Homebrew should be using it regularly. This is partly because you won't be a good maintainer unless you can put yourself in the shoes of our users but also because you may decide to stop using Homebrew and at that point you should also decide to not be a maintainer and find other things to work on.
+
+# 2. No Guilt About Leaving
+
+All maintainers can stop working on Homebrew at any time without any guilt or explanation (like a job). We may still ask for your help with questions after you leave but you are under no obligation to answer them. Like a job, if you create a big mess and then leave you still have no obligations but we may think less of you (or, realistically, probably just revert the problematic work). Like a job, you should probably take a break from Homebrew at least a few times a year.
+
+This also means contributors should be consumers. If an owner finds they
+are not using a project in the real-world, they should reconsider their
+involvement with the project.
+
+# 3. Maintainers > Users
```

Comment:
As delightful as this title is, perhaps there's a less provocative way to say the same thing.